server/belga/search_providers.py

```diff
@@ -26,6 +26,8 @@ def get_text(value, strip_html=True):
     try:
         text = value.strip()
         if strip_html:
+            # Preserve ampersand character
+            text = text.replace("&", "&amp;")
```

Comment:
I wonder if we shouldn't rather use some html.escape, but anyhow would be good to test this

Reply:
Yep, I will add a test case for it.